internal/projects/deleter_test.go

```diff
@@ -51,6 +51,12 @@ func TestDeleteProjectOneProjectWithNoParents(t *testing.T) {
 		}, nil)
 	mockStore.EXPECT().ListProvidersByProjectID(gomock.Any(), []uuid.UUID{proj}).
 		Return([]db.Provider{}, nil)
+	mockStore.EXPECT().CountProfilesByProjectID(gomock.Any(), proj).
+		Return(int64(0), nil)
+	mockStore.EXPECT().CountRepositoriesByProjectID(gomock.Any(), proj).
+		Return(int64(0), nil)
+	mockStore.EXPECT().GetEntitlementsByProjectID(gomock.Any(), proj).
+		Return([]db.Entitlement{}, nil)
```

Comment:
Can you add at least one test where you verify that these are getting added to the telemetry context?